python-sdk/dev/scripts/pre_commit_context_typing_compat.py

```diff
@@ -27,14 +27,16 @@ def visit_ImportFrom(self, node: ImportFrom) -> None:
             return
 
         for alias in node.names:
-            if f"{node.module}.{alias.name}" == "airflow.utils.context.Context":
-                if not self.has_incompatible_context_imports:
-                    self.has_incompatible_context_imports = True
+            if (
+                f"{node.module}.{alias.name}" == "airflow.utils.context.Context"
+                and not self.has_incompatible_context_imports
+            ):
+                self.has_incompatible_context_imports = True
 
 
 def get_all_provider_files() -> list[Path]:
     """Retrieve all eligible provider module files."""
-    provider_files = []
+    _provider_files = []
```

Comment:
@pankajastro What was the issue here?

Reply:
define in outer scope https://github.com/astronomer/astro-sdk/pull/1208/files#diff-93af335d96ed6e2ad0b9486f907098f5a5637f741d6d244833b226c4ed84fc29R67